ui/src/components/Layouts/InputGrid/InputGrid.tsx

```diff
@@ -11,13 +11,19 @@ const InputGrid: FC<InputGridProps> = ({ children, className }) => {
 
 	return (
 		<Row gutter={[16, 0]} className={className}>
-			{childElements.map((child, index) => (
-				<Col key={index} xs={24} sm={24} md={24} lg={12}>
+			{childElements.map((child) => (
+				<Col key={getKey(child)} xs={24} sm={24} md={24} lg={12}>
 					{child}
 				</Col>
 			))}
 		</Row>
 	);
 };
 
+// Function to generate unique keys based on child content
+const getKey = (child: React.ReactNode): string => {
```

Comment:
@panagiotisbellias can you please share more about this approach. For example any articles, docs etc

Reply:
In this project we don't write helper functions inside a component.

Reply:
> @panagiotisbellias can you please share more about this approach. For example any articles, docs etc

@lifeparticle To avoid using array indices as keys, I generated unique keys for each child element using a unique identifier associated with each child element. In the getKey function, I generated the key based on the content of each child element. This ensures that each child has a unique key without relying on array indices. Probably this needs testing too when above information is provided.



> In this project we don't write helper functions inside a component.

I'm sorry, didn't notice. Please confirm that the correct location for the helper function after testing it should be `ui/src/utils/helper-functions`

Thanks,
Panagiotis

Reply:
@panagiotisbellias Thank you for the explanation. Can you please check the official doco here, https://react.dev/learn/rendering-lists

I'm not sure if this is the right approach to generate a key.

Please put it in the same directory `./helper.ts`

Thanks

Reply:
@lifeparticle Indeed this is not the right approach to generate a key. This has to be done per case according to what attributes each list's object have like id, uuid etc.

I proceed with that